sdk/template-dpg/Azure.ServiceTemplate.Template/tests/Samples/TemplateClientSamples.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ServiceTemplate.Template.Tests.Samples
+{
+    public class TemplateClientSamples : SamplesBase<TemplateClientTestEnvironment>
```

Comment:
I don't see other data plane libraries that have this file under <library>/tests/samples.  What motivates this?

Reply:
Current our package structure contains sdk/<service>/tests/samples and put all the sample code here, and will add md in sdk/<service>/Samples directory to link to it.
e.g. https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/anomalydetector/Azure.AI.AnomalyDetector